```
Traceback (most recent call last):
  File "/Users/gabrielunderwood/repos/labelbox-python/test.py", line 9, in <module>
    test = client.create_prompt_response_generation_project(
  File "/Users/gabrielunderwood/repos/labelbox-python/libs/labelbox/src/labelbox/client.py", line 955, in create_prompt_response_generation_project
    return self._create_project(**kwargs)
  File "/Users/gabrielunderwood/repos/labelbox-python/libs/labelbox/src/labelbox/client.py", line 1071, in _create_project
    return self._create(Entity.Project, params, extra_params)
  File "/Users/gabrielunderwood/repos/labelbox-python/libs/labelbox/src/labelbox/client.py", line 640, in _create
    res = self.execute(query_string, params, raise_return_resource_not_found=True)
  File "/Users/gabrielunderwood/repos/labelbox-python/.venv/lib/python3.8/site-packages/google/api_core/retry/retry_unary.py", line 293, in retry_wrapped_func
    return retry_target(
  File "/Users/gabrielunderwood/repos/labelbox-python/.venv/lib/python3.8/site-packages/google/api_core/retry/retry_unary.py", line 153, in retry_target
    _retry_error_helper(
  File "/Users/gabrielunderwood/repos/labelbox-python/.venv/lib/python3.8/site-packages/google/api_core/retry/retry_base.py", line 212, in _retry_error_helper
    raise final_exc from source_exc
  File "/Users/gabrielunderwood/repos/labelbox-python/.venv/lib/python3.8/site-packages/google/api_core/retry/retry_unary.py", line 144, in retry_target
    result = target()
  File "/Users/gabrielunderwood/repos/labelbox-python/libs/labelbox/src/labelbox/client.py", line 309, in execute
    raise labelbox.exceptions.ResourceNotFoundError(
labelbox.exceptions.ResourceNotFoundError: Failed to find dataset: Demo prompt response dataset('Failed to find dataset: Demo prompt response dataset', None)
```

--- a/libs/labelbox/src/labelbox/client.py
+++ b/libs/labelbox/src/labelbox/client.py
@@ -637,7 +637,7 @@ class Client:
 
         data = {**data, **extra_params}
         query_string, params = query.create(db_object_type, data)
-        res = self.execute(query_string, params)
+        res = self.execute(query_string, params, raise_return_resource_not_found=True)
 
         if not res:
             raise labelbox.exceptions.LabelboxError("Failed to create %s" %

